MaxText/train.py

```diff
@@ -41,9 +41,11 @@
 import optimizers
```

Comment:
It's hard to tell what the actual change in this file is other than formatting. If it does need to be reformatted, can you check that in with a separate PR first?

Reply:
Yeah I accidentally "save" this file so it's reformatted with 4 spaces for indentation. Will fix.

Reply:
Not showing diff for this file now as the styles are consistent. Resolving this.